executor/Dockerfile.executor

```diff
@@ -17,6 +17,7 @@ COPY api/ api/
 COPY predictor/ predictor/
 COPY logger/ logger/
 COPY k8s/ k8s/
+COPY certs/ certs/
```

Comment:
:question: Given the change to `.gitignore`, should this line be here or was it just for local testing?

Reply:
Ah yes it was for local testing, I'll remove it .